components/CrawlStatusMonitor.tsx

```
@@ -0,0 +1,266 @@
+                   <div className="flex items-center space-x-2 flex-grow min-w-0">
+                     {/* Render checkbox only if the URL is crawlable */}
+                     {isCrawlable ? (
+                       <input
+                         type="checkbox"
+                         className="form-checkbox h-4 w-4 text-blue-600 bg-gray-700 border-gray-600 rounded focus:ring-blue-500 flex-shrink-0"
+                         checked={selectedUrls.has(url)} // Use prop
+                         onChange={() => handleCheckboxChange(url)}
+                         aria-label={`Select URL ${url}`}
+                       />
+                     ) : (
+                       <div className="w-4 h-4 flex-shrink-0"></div> // Placeholder for alignment
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)